package.json

```diff
@@ -42,11 +54,14 @@
   "devDependencies": {
     "@types/jest": "^22.2.3",
     "@types/supertest": "^2.0.4",
-    "husky": "^0.14.3",
+    "husky": "^1.0.0-rc.6",
     "jest": "^22.4.3",
     "lerna": "^2.11.0",
+    "lint-staged": "^7.1.1",
     "mock-req": "^0.2.0",
+    "prettier": "1.12.1",
     "rimraf": "^2.6.2",
+    "rxjs-compat": "^6.1.0",
```

Comment:
Why do we need `rxjs-compat` - it is used only for compatibility reasons with old RxJS api? 🤔

Reply:
@JozefFlakus Husky is in `rc.6` and uses old RxJS api still :C we require that until `husky` is in `v1` version released

Reply:
But do we need the newest version of Husky? 🤔

Reply:
@JozefFlakus This version was recommended on their GH page

Reply:
I would still prefer to use the old API - it works, stable and doesn't require old stuff like `rxjs-compat`. I'm open to migrate to the newest version when it will move outside `RC` stage.

Reply:
BTW. I don't see there any RxJS dependency https://github.com/typicode/husky/blob/dev/package.json

Reply:
ok, doors opened